subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -334,31 +337,34 @@ void radio_pkt_configure(uint8_t bits_len, uint8_t max_len, uint8_t flags)
 	/* To use same Data Channel PDU structure with nRF5 specific overhead
 	 * byte, include the S1 field in radio packet configuration.
 	 */
-	if (dc) {
+	if (pdu_type == RADIO_PKT_CONF_PDU_TYPE_DC) {
 		extra |= (RADIO_PCNF0_S1INCL_Include <<
 			  RADIO_PCNF0_S1INCL_Pos) & RADIO_PCNF0_S1INCL_Msk;
+#if defined(CONFIG_BT_CTLR_DF)
+		if (RADIO_PKT_CONF_CTE_GET(flags) == RADIO_PKT_CONF_CTE_ENABLED) {
+			bits_s1 = 8U;
```

Comment:
What does `_s1` mean in this context? 

And what does the value `8U` mean?

Reply:
`S1` is a name of a byte used in Radii peripheral SOCs documentation. 
`bits` simitlar to `bits_len` means nubmer of bits in the `S1` byte. Hence 8U for 8 bits.